### DISCUSSION TITLE
Very long pause when I start Neurodesktop on a NECTAR instance

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#128] [Q&A] [answered]
Hello,

When starting Neurodesktop on a NECTAR instance, it prints right on the start:
**usermod: no changes**
and then there is a very long pause (a few minutes) until it prints:
**=====**
**Starting SSH server**
*Starting OpenBSD Secure Shell server sshd
**======**
After that, it prints the rest of the stuff very quickly, until it is ready to use displaying:
**guacd[629]: INFO:	Listening on host 127.0.0.1, port 4822**

What is this pause about? Does it happen on all platforms? Notice that I experience the pause even when the vnmd/neurodesktop image is already downloaded into the NECTAR instance.

If this pause is an inevitable part of neurodesktop's stratup, I suggest to print a message such as "Loading. Please wait ..." before the pause. Otherwise, people may think something is stuck or not working (I thought so). 

Oren

P.S.
The exact command I use is:
sudo docker run   --shm-size=1gb -it --privileged --name neurodesktop   -v ~/neurodesktop-storage:/neurodesktop-storage   -e HOST_UID="$(id -u)" -e HOST_GID="$(id -g)"   -p 8080:8080 -h neurodesktop-20210929   vnmd/neurodesktop:20210929

**Comment by stebo85** [comment, marked as answer]
Dear @civier - This looks like a bug on Nectar and it would be interseting to see what's happening there. As there is a few minutes delay (can you quantify how much exactly?) it looks like a port issue that times out? @aswinnarayanan any ideas? I have seen a similar behaviour when I run zoom and screen share but was assuming that it's some bandwith issue. Converting to an issue so we can follow up on this.

**Reply by aswinnarayanan** [reply]
Fixed https://github.com/NeuroDesk/neurodesktop/commit/85c97be7d845a6430516a98e97f4a905e2180f54
Changing the gid of all files and folder in home dir was very slow on some filesystems due to the large miniconda dir.
Switched to just changing the home dir only now

**Reply by stebo85** [reply]
Wow! That's a good find and great fix. Miniconda will also go away soon from the main image and move to cvmfs :)
